Xamarin.Forms.Build.Tasks/xlf/ErrorMessages.cs.xlf

```diff
@@ -0,0 +1,182 @@
+        <note>0 is type name</note>
+      </trans-unit>
+      <trans-unit id="Conversion">
+        <source>Cannot convert value "{0}" to "{1}".</source>
+        <target state="new">Cannot convert value "{0}" to "{1}".</target>
+        <note>0 is a value, 1 is a type</note>
+      </trans-unit>
+      <trans-unit id="EnumValueMissing">
+        <source>Enum value not found for "{0}".</source>
+        <target state="new">Enum value not found for "{0}".</target>
+        <note>0 is enum value</note>
+      </trans-unit>
+      <trans-unit id="InvalidXaml">
+        <source>Invalid Xaml "{0}".</source>
+        <target state="new">Invalid Xaml "{0}".</target>
+        <note>0 is the invalid xaml snippet</note>
+      </trans-unit>
+      <trans-unit id="MarkupNotClosed">
+        <source>Markup expression not closed.</source>
+        <target state="new">Markup expression not closed.</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="MarkupParsingFailed">
+        <source>Error while parsing markup expression.</source>
+        <target state="new">Error while parsing markup expression.</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="MemberResolution">
+        <source>No property, BindableProperty, or event found for "{0}", or mismatching type between value and property.</source>
+        <target state="new">No property, BindableProperty, or event found for "{0}", or mismatching type between value and property.</target>
+        <note>0 is member name</note>
+      </trans-unit>
+      <trans-unit id="MethodStaticMissing">
+        <source>No static method found for "{0}::{1} ({2})".</source>
+        <target state="new">No static method found for "{0}::{1} ({2})".</target>
+        <note>0 is type name, 1 is methodname, 2 is type arg list</note>
+      </trans-unit>
+      <trans-unit id="MissingEventHandler">
+        <source>EventHandler "{0}" with correct signature not found in type "{1}".</source>
+        <target state="new">EventHandler "{0}" with correct signature not found in type "{1}".</target>
+        <note>0 is the handler name, 1 is the declaring type</note>
+      </trans-unit>
+      <trans-unit id="NamescopeDuplicate">
+        <source>An element with the name "{0}" already exists in this NameScope.</source>
+        <target state="new">An element with the name "{0}" already exists in this NameScope.</target>
+        <note>0 is the duplicated key</note>
+      </trans-unit>
+      <trans-unit id="PropertyMissing">
+        <source>Missing mandatory property "{0}" on "{1}".</source>
+        <target state="new">Missing mandatory property "{0}" on "{1}".</target>
+        <note>0 is a property name, 1 is a type</note>
+      </trans-unit>
+      <trans-unit id="PropertyResolution">
+        <source>Cannot resolve property "{0}" on type "{1} (property missing, or missing accessors)".</source>
```

Comment:
Comma check @hartez @samhouts ?

Reply:
burn it 🔥

Reply:
Looks fine either way to me, if there's a vote to burn it then light it up.